e2e/test/MultiplexingOverAmqp.cs

```diff
@@ -15,6 +15,8 @@ public static class MultiplexingOverAmqp
         public const int MuxWithoutPoolingPoolSize = 1; // For enabling multiplexing without pooling, the pool size needs to be set to 1
         public const int MuxWithPoolingDevicesCount = 4;
         public const int MuxWithPoolingPoolSize = 2;
+        public const int maxTestRunCount = 5;
+        public const float testSuccessRate = 0.8f; // 4 out of 5 test runs should pass (even after accounting for network instability issues).
```

Comment:
Nit: 
`We expect network issues during test execution [ ... ] `.

Please explain the reason we need to do this kind of strict assertion that is sensitive to noise (network failing during test execution).